<!doctype html>
<html class="no-js" lang="en-gb">

<head>
  <meta charset="utf-8">
  <title>Zebrafish Genome Literacy Workshop 2023 | Busch Lab</title>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="https://www.buschlab.org/css/normalize.css">
  <link rel="stylesheet" href="https://www.buschlab.org/css/main.css?20191007">
  <link href="https://fonts.googleapis.com/css?family=Roboto:400,700%7CUbuntu+Condensed" rel="stylesheet">
  <link rel="apple-touch-icon" sizes="180x180" href="https://www.buschlab.org/apple-touch-icon.png">
  <link rel="icon" type="image/png" sizes="32x32" href="https://www.buschlab.org/favicon-32x32.png">
  <link rel="icon" type="image/png" sizes="16x16" href="https://www.buschlab.org/favicon-16x16.png">
  <link rel="manifest" href="https://www.buschlab.org/site.webmanifest">
  <link rel="mask-icon" href="https://www.buschlab.org/safari-pinned-tab.svg" color="#5bbad5">
  <meta name="msapplication-TileColor" content="#2d89ef">
  <meta name="theme-color" content="#ffffff">
  <style>
    .answer { color: red; }
  </style>
</head>
<body>

  <div id="container">
    <header>
      <a href="https://www.buschlab.org/"><img src="https://www.buschlab.org/img/logo.svg" alt="Busch Lab" /></a>
      <nav>
        <ul>
          <li>
            <a href="https://www.buschlab.org/">Home</a>
          </li>



          <li>
            <a href="https://www.buschlab.org/contact/">Contact</a>
          </li>



          <li>
            <a href="https://www.buschlab.org/news/">News</a>
          </li>



          <li>
            <a href="https://www.buschlab.org/publications/">Publications</a>
          </li>



          <li>
            <a href="https://www.buschlab.org/research/">Research</a>
          </li>



          <li>
            <a href="https://www.buschlab.org/resources/">Resources</a>
          </li>



          <li>
            <a href="https://www.buschlab.org/software/">Software</a>
          </li>



          <li>
            <a href="https://www.buschlab.org/team/">Team</a>
          </li>


        </ul>
      </nav>
    </header>
    <main>
      <h1>Zebrafish Genome Literacy Workshop 2023</h1>

<h2>Exercise 3 - CRISPR design</h2>

<p>Imagine you are planning an experiment to knock out the tfap2a gene using CRISPR/Cas9.</p>

<p>Find the Gene page for tfap2a. How many different transcripts are there?</p>

<p class="answer">6.</p>

<p>We want to target exons that appear in all transcripts. Find the stable IDs of exons which are present in all transcripts of tfap2a.</p>

<ul class="answer">
<li>ENSDARE00000597910</li>
<li>ENSDARE00000597893</li>
<li>ENSDARE00000597878</li>
<li>ENSDARE00000597872</li>
<li>ENSDARE00000597861</li>
</ul>

<p class="answer">Exons may look the same on the graphical view on the Gene page, but may have slightly different start and end sites. You can use the Exons page for each transcript to compare the exons. Exons with identical start and end sites will have the same exon ID.</p>

<p>Use Biomart to download the sequence for these exons. Use the exon IDs you collected above to download the exons sequences with 10 bp of intron sequence on either side. The 10 bp allows you to find CRISPR sites that overlap a small portion of the intron, but still have their cut-site within the exon.</p>

<p class="answer">Use the &quot;Input external references ID list&quot; filter and select &quot;Exon ID&quot; from the dropdown menu. You can use the Upstream flank/Downstream flank attributes to do this. Some tools don't like non-alphanumeric characters in FASTA files, so just have the exon id in the sequence headers</p>

<p class="answer">Attributes</p>
<ul class="answer">
<li>Exon stable ID</li>
<li>Exon sequences</li>
<li>Upstream flank [10]</li>
<li>Downstream flank [10]</li>
</ul>

<p>Use these sequences to select sgRNA binding sites. Use any CRISPR design tool you want (e.g. <a href="https://eu.idtdna.com/site/order/designtool/index/CRISPR_CUSTOM">https://eu.idtdna.com/site/order/designtool/index/CRISPR_CUSTOM</a> or <a href="https://www.crisprscan.org/sequence/">https://www.crisprscan.org/sequence/</a>).</p>

<p><strong>Note:</strong> For the IDT tool you will need to edit the FASTA file to remove characters from the sequence names that the tool doesn't accept. The easiest thing to do is to only select the exon ID for the sequence header.</p>

<p>Select several guide RNA sequences.  Visualise the gRNA sequences using Ensembl BLAST. Choose BLASTN with &quot;Normal&quot; search sensitivity. Check that the cut-site will be inside the exon and not too close to the splice acceptor/donor. The cut-site is 6 bp from the end of the sgRNA sequence including PAM (e.g. NNNNNNNNNNNNNNNNN|NNNNGG).</p>

<p>Check for off-targets in other tfap2 genes. If your CRISPR design tool of choice doesn't do off-target prediction, you could use BLASTN and change the search sensitiivty to &quot;Short sequences&quot; to find sequences similar to the guide sequence.</p>

    </main>
  </div>

  <footer>
    <ul>
      <li>
        <a href="https://twitter.com/busch_lab?ref_src=twsrc%5Etfw" class="twitter-follow-button" data-show-screen-name="false" data-lang="en" data-show-count="false">Follow @busch_lab</a>
        <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
      </li>
    </ul>
    <ul>
      <li>&copy; Busch Lab 2024</li>
    </ul>
  </footer>
  <script>
    window.ga = function () { ga.q.push(arguments) }; ga.q = []; ga.l = +new Date;
    ga('create', 'UA-142839217-3', 'auto'); ga('send', 'pageview')
  </script>
  <script src="https://www.google-analytics.com/analytics.js" async defer></script>
</body>

</html>
